src/it/java/io/deephaven/benchmark/tests/standard/updateby/RollingComboTest.java

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2022-2023 Deephaven Data Labs and Patent Pending */
+package io.deephaven.benchmark.tests.standard.updateby;
+
+import org.junit.jupiter.api.*;
+import io.deephaven.benchmark.tests.standard.StandardTestRunner;
+
+/**
+ * Standard tests for the updateBy table operation. Combines multiple rolling operations
+ */
+public class RollingComboTest {
```

Comment:
Thanks for adding this, this looks good.  When you get a chance, I would also like a test for only cumulative operators (cum_sum, cum_min, ema etc.).  There are some optimizations possible when we are computing non-rolling operators that will be great to measure. 

One more suggestion for a test, a wider UpdateBy test that includes both cumulative and rolling operators would be helpful.

Reply:
Added https://github.com/deephaven/benchmark/issues/64 to cover this